x/gov/types/keys.go

```diff
@@ -42,6 +42,7 @@ var (
 	ActiveProposalQueuePrefix   = []byte{0x01}
 	InactiveProposalQueuePrefix = []byte{0x02}
 	ProposalIDKey               = []byte{0x03}
+	ExpeditedProposalsKeyPrefix = []byte{0x04}
```

Comment:
This key isn't being used I think it's good to delete this for now unless we have future plans to use this
```suggestion
```

Reply:
Nice catch. Originally, was going to implement this via a second index but found a workaround